src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -386,6 +389,14 @@ def api_version(self, value):
         """Set the connection api version."""
         self.configs["api_version"] = value
 
+    @property
+    def token(self):
+        """Return the connection token."""
+        if isinstance(self.token_provider, str):
```

Comment:
It is not that easy to understand that token provider could be a string?

Reply:
Agree, but we need to provide an easy way for user to provide a static token for a connection. What's your suggestion ?

Reply:
I think we can have a StaticTokenProvider instead of put a string as a token provider?

Reply:
Get your point, I think that's a good idea.